src/onegov/search/cli.py

```diff
@@ -39,7 +53,9 @@ def psql_index_status(app: 'Framework') -> None:
             success = 3
             continue
 
-        q = session.query(func.count(model.fts_idx))
+        count_column = get_non_nullable_columns(model)[0]
+        q = session.query(func.count(count_column))
```

Comment:
Have you tried:

```suggestion
        q = session.query(model).with_entities(func.count(text('*'))
```

That seems a lot more robust to me than inspecting the table, that should generate `SELECT count(*) FROM ...`

Reply:
Will do so

Reply:
Works as expected, just commited